m4_sessions/1708-inning/group_two.md

```diff
@@ -1,9 +1,45 @@
   # Group Two
 
 ### Session Title:
+* The Psychology of UX/UI
 
 ### Session Outcomes: 
+What do we want people attending the session to learn or walk away with at the end of the session?
+
+#### Education
+* Understanding the difference of UX and UI
+* The importance of NOT ignoring UX/UI (why *everyone* should care about it)
+* Understanding the emotions of UX/UI
+
+#### Simplicity
+* The power of simplicity
+* Understanding design patterns in UX/UI (and recognizing/understanding bad or anti-patterns)
+
+#### In Your Career
+* Understanding proper usage of micro-animations (and NOT overusing the BLINK tag) - live examples/demo
+* How to incorporate UX/UI thinking into your career as a developer
+* Being more actively aware of usability
 
 ### Session Outline:
+Intro to UX/UI (10 minutes)
+- need to come up with what this looks like
+Design Thinking Video (10 minutes or less)
+- need to find one or two great videos
+Leader Group Exercise (10 minutes)
+- what we think is great/bad design with examples
+Small Group Exercise (10 minutes of research/talking, 5 minutes of sharing)
+- need to come up with "what" the groups will be doing
+Share principles of Simplicity, w/ slide show (10 minutes)
+Career Prep (15 minutes)
```

Comment:
Really love the flow to this session, though it might feel like breadth vs. depth.  As you refine, you may want to go deeper into one area and give more time for that conversation.  You could always have more resources and create a slack channel for future conversations.

Reply:
Thanks for the feedback. We voted to allow more time for the small group exercise and move Career Prep to a list of resources for people to follow up on afterward in their own time.